app/views/posts/show.html.erb

```diff
@@ -1,2 +1,21 @@
-<h1>Posts</h1>
-<p>Shows all the posts by a particular user</p>
\ No newline at end of file
+<div class="full_post">
+<div class="post_body">
+  <div class="post_header">
+    <h4>Post by: <%= @post.author.name %></h4>
+    <p>
+      Comments: <%= @post.comments_counter %> 
+      Likes: <%= @post.likes_counter %>
+    </p>
+  </div>
+  <p class="post-text"><%= @post.text %></p>
+</div>
+  <div class="post-comments">
+    <% if @post.comments.empty? %>
+    <p>Add comment</p>
+    <% else %>
+    <% @post.recent_comments.each do |comment| %>
+      <p><%= comment.author.name %>: <%= comment.text %></p>
+    <%end%>
+    <%end%>
+</div>
```

Comment:
- [ ] Great work styling the pages to look like the wireframes however the user post shows is not styled according to the wireframes, kindly make sure to fix this so your views is showing as expected

|Posts/show|wireframe|
|-|-|
|![Screenshot_20230309_145104](https://user-images.githubusercontent.com/25227022/224044869-2d1d5c46-8ee2-48f3-ac69-7cb392331867.png)|![Screenshot_20230309_145156](https://user-images.githubusercontent.com/25227022/224045011-2d3e6d65-af73-4e0c-a3a5-c91cc221b11e.png)|